src/prompt.ts

```diff
@@ -1,6 +1,6 @@
 import * as readline from "readline";
 import chalk from "chalk";
-import stripAnsi from "strip-ansi";
+import stripAnsi = require("strip-ansi");

```

Comment:
Does this compile? This is CJS syntax in ESM but maybe it works?

Reply:
It's a feature of `module:nodenext`, @weswigham added support for transforming this to use `createRequire`.